RESOURCES/INTHEWILD.md

```diff
@@ -112,7 +112,7 @@ Join our growing community!
 - [Showmax](https://showmax.com) [@bobek]
 - [TechAudit](https://www.techaudit.info) [@ETselikov]
 - [Tenable](https://www.tenable.com) [@dflionis]
-- [Tentacle](https://tentaclecmi.com) [@jdclarke5]
+- [Tentacle](https://www.linkedin.com/company/tentacle-cmi/) [@jdclarke5]
```

Comment:
Just FYI, the link checker is non-blocking. I'm wondering if these guys went out of business or something. The site's been down for about a week thus far, and I didn't get a response from them on Slack.